CMakeLists.txt

```diff
@@ -207,6 +207,19 @@ cmake_dependent_option(
 option(USE_TBB "Use TBB" OFF)
 option(ONNX_ML "Enable traditional ONNX ML API." ON)
 
+# Linux distributions do not want too many embedded sources
+option(USE_SYSTEM_LIBS "Use system-provided libraries." OFF)
+option(USE_SYSTEM_CPUINFO "Use system-provided cpuinfo." OFF)
+option(USE_SYSTEM_SLEEF "Use system-provided sleef." OFF)
+if(DEFINED ENV{USE_SYSTEM_LIBS})
```

Comment:
I don't think you need the explicit env var check here; setup.py will handle converting environment variables into cmake flags.

Reply:
This env check is convenient for my local testing. I can try to add replacement code to setup.py for the next PR.

Reply:
@cdluminate You don't need to add anything to `setup.py`. Simply remove this `if` block and it'll be automatically passed to CMake in https://github.com/pytorch/pytorch/blob/438aed63a143103180eeff3e8991413e74569563/tools/setup_helpers/cmake.py#L251-L261